### PR TITLE
Update Tresorit

### PR DESCRIPTION
**Pull request by leggomyinfo**
Tresorit requires enabling at least two methods.

### CODE REVIEW

**Comment by phallobst** [comment]
According to the doc page, this is not actually required. It says
> If you have multiple verification methods set up, ...
> ...if you set up a secondary option as a backup.

Is the documentation out of date, i.e. do you have first-hand information here?

**Comment by leggomyinfo** [comment]
> According to the doc page, this is not actually required. It says
> 
> > If you have multiple verification methods set up, ...
> > ...if you set up a secondary option as a backup.
> 
> Is the documentation out of date, i.e. do you have first-hand information here?

@phallobst It's actually in the documentation, albeit hard to spot: [https://support.tresorit.com/hc/article_attachments/360003584614/primary_2x.gif](https://support.tresorit.com/hc/article_attachments/360003584614/primary_2x.gif)

**Comment by phallobst** [comment]
I did see that, but it's still contradicting the text. Either the text or the GIF is out of date, I guess.

Edit: I tried it out myself. The user actually needs to select at least two methods, so the GIF is up-to-date and the text is not. Removing the second-to-last option disables 2FA altogether.

**Comment by leggomyinfo** [comment]
The text is out-of-date. I have a Tresorit account and confirmed two are required.

File: _data/backup.yml [hunk 1]
```diff
@@ -225,6 +225,7 @@ websites:
       - email
       - totp
     doc: https://support.tresorit.com/hc/en-us/articles/216114527
+    exception: Requires enabling at least two methods.
```
Comment (Carlgo11) [inline _data/backup.yml]:
```suggestion
    exception: "Requires enabling at least two 2FA methods."
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Review by kmpoppe — APPROVED** [review]

**Pull request merged**